react-components/src/architecture/base/commands/BaseCommand.ts

```diff
@@ -0,0 +1,100 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+/* eslint-disable @typescript-eslint/class-literal-property-style */
```

Comment:
unnecessary — don't notice any difference when I remove this

Reply:
No, it gives lint errors in all virtual functions that return a constant.

Reply:
If we need this rule disabled, we can disable it in the `eslintrc.cjs`